doc/source/index.rst

```diff
@@ -1,60 +1,79 @@
 .. _index:
 
-Welcome to the **kOS** Documentation Website!
-=============================================
+kOS: Kerbal Operating System
+============================
 
-**kOS** is a scriptable autopilot modification for **Kerbal Space Program**. It allows you write small programs that automate specific tasks.
+==================================== ======================== ===================================
+:ref:`Full Documentation <contents>` :ref:`Download <obtain>` :ref:`Getting started <quickstart>`
+==================================== ======================== ===================================
 
-.. toctree::
-    :includehidden:
-    :titlesonly:
-    :maxdepth: 5
+::
 
-    Home <self>
-    Downloads and Links <downloads_links>
-    Tutorials <tutorials>
-    Community Example Library <library>
-    General <general>
-    Language <language>
-    Mathematics <math>
-    Commands <commands>
-    Structures <structures>
-    Addons <addons>
-    Contribute <contribute>
-    Changes <changes>
-    About <about>
+    PRINT "Hello World.".
+    PRINT "These are the documents for Kerbal Operating System.".
 
-Introduction to **kOS** and **KerboScript**
-===========================================
 
-**KerboScript** is the language used to program the CPU device attached to your vessel and **kOS** is the operating system that interprets the code you write. The program can be as simple as printing the current altitude of the vessel and as complicated as a six-axis autopilot controller taking your vessel from the launchpad to Duna and back! With **kOS**, the sky is *not* the limit.
+The do-it-yourself autopilot
+----------------------------
 
-This mod *is* compatible with `RemoteTech`_, you just have to make sure you copy the program onto the local CPU before it goes out of range of KSC.
+**kOS**, or Kerbal Operating System, is a community-supported mod for the popular game
+ `Kerbal Space Program <https://kerbalspaceprogram.com/>`_
```

Comment:
Was not intentional.  I cut the line because the link was long, but it was meant to merge together into one line on output, as usually happens.  I didn't remove the space when I inserted the 'enter' between the terms.